scripts/flow/v10/chatListener.js

```diff
@@ -0,0 +1,21 @@
+import {getFlowInfo} from "./messageParser.js";
+import {processFlowInfo} from "../common.js";
+
+
+const _onReady = () => {
+    if (foundry.utils.isNewerVersion(game.version, "11")) return;
```

Comment:
This is v10 only. The v11 system no longer stores information in chat messages, so a different implementation is used.